tests/test_intercept.js

```diff
@@ -220,6 +220,25 @@ test("reply should send correct statusCode with array-notation and without body"
   req.end();
 });
 
+test("adds a leading slash when the path doesn't include one", function(t) {
+  t.plan(1);
+
+  var scope = nock("http://www.google.com")
```

Comment:
Mocked request tests should use a fake hostname (see #1351). Could you change this from `www.google.com` to `example.test`?

Reply:
Sure, will do.